WEBPAGE/main.py

```
@@ -0,0 +1,911 @@
+f.setLevel(logging.INFO)
+f.setFormatter(formatter)
+c.setLevel(logging.INFO)
+c.setFormatter(formatter)
+
+logger.addHandler(c)
+logger.addHandler(f)
+
+
+logger.info("App initializing")
+app = Flask(__name__)
+app.secret_key = 'user_id'
```

Ruff (0.14.7)

Possible hardcoded password assigned to: "secret_key"

(S105)